grumphp.yml

```diff
@@ -1,13 +1,8 @@
-parameters:
-  git_hook_variables:
-    EXEC_GRUMPHP_COMMAND: docker-compose exec -T php
+grumphp:
   extensions:
     - GrumphpDrupalCheck\ExtensionLoader
-  ascii:
-    failed: vendor/vijaycs85/drupal-quality-checker/resources/ascii/grumpy.txt
-    succeeded: vendor/vijaycs85/drupal-quality-checker/resources/ascii/happy.txt
-  git_dir: .
-  bin_dir: vendor/bin
+  hooks_dir: ~
+  hooks_preset: local
```

Comment:
In the documentation says `Presets are only used when you did NOT specify a custom hooks_dir`
https://github.com/phpro/grumphp/blob/master/doc/parameters.md

Is this line necessary?

Reply:
Not really, that's true. Plus, local is the default value. Fixing that.